servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSConsts.java

```diff
@@ -229,5 +231,15 @@
     public static final int STRING_BLDR_SIZE_DEFAULT = 512;
 
     public static final String ZMS_JSON_PARSER_ERROR_RESPONSE = "{\"code\":400,\"message\":\"Invalid Object: checkout https://github.com/yahoo/athenz/core/zms for object defintions\"}";
+
+    public static final String NOTIFICATION_TYPE_MEMBERSHIP_APPROVAL = "MEMBERSHIP_APPROVAL";
```

Comment:
lines 235 - 240: let's move these as public strings in the Notification interface class. This way the implementation can just reference and use them directly instead of having their own values and possibly misspelling, etc.